backend/drive/drive.go

```diff
@@ -2578,125 +2588,18 @@ func (f *Fs) ChangeNotify(ctx context.Context, notifyFunc func(string, fs.EntryT
 				if pollInterval != 0 {
 					ticker = time.NewTicker(pollInterval)
 					tickerC = ticker.C
+					// TODO: Make the initial buffer configurable
```

Comment:
Is this looking for changes in the past? This seems to say 10 * the pollInterval which sounds a lot. I would have thought a few seconds is all we need?

Reply:
This is the initial buffer period for any changes that may get missed while doing directory listings before the change notify service has started, hence the larger initial buffer. I am not sure if that's even possible or how large of an interval that could be.

Let me know if you want me to remove that or make it shorter.